src/main/java/org/fundacionjala/sfdc/util/CommonActions.java

```diff
@@ -39,17 +40,26 @@ private CommonActions() {
      * @return the element.
      */
     public static WebElement getElement(final WebElement element) {
+        waitTime(1);
         WAITER.until(ExpectedConditions.visibilityOf(element));
         return element;
     }
 
+    /**
+     * This method generates a wait for a fixed time.
+     *
+     * @param time .
+     */
+    public static void waitTime(int time) {
+        WEB_DRIVER.manage().timeouts().implicitlyWait(time, TimeUnit.SECONDS);
```

Comment:
@AT-06/at-06 
This change will have problems in the future.
if you are changing the implicit wait or explicit wait 
you should ensure to restore by the default value.

Consider using a try - finally